src/components/TeacherComponents/OnboardingForm/OnboardingForm.tsx

```diff
@@ -51,17 +54,27 @@ const OnboardingForm = ({
   const utmParams = useUtmParams();
   const { posthogDistinctId } = useAnalytics();
   const { user } = useUser();
+  const [submitError, setSubmitError] = useState<string | null>(null);
```

Comment:
Storing this as local state so that the form can be re-submitted should the onboarding request fail.